MSCL/source/mscl/MicroStrain/MIP/MipNode_Impl.cpp

```diff
@@ -2191,6 +2191,29 @@ namespace mscl
         return r.parseResponse(doCommand(r, ActivationCode::buildCommand_get()));
     }
 
+    EventSupportInfo MipNode_Impl::getEventInfo(const EventSupportInfo::Query query) const
+    {
+        const MipFieldValues response = get(MipTypes::Command::CMD_EVENT_SUPPORT,
+                    { Value::UINT8(static_cast<uint8>(query)) }
+        );
+
+        EventSupportInfo info{};
+
+        info.query = static_cast<EventSupportInfo::Query>(response[0].as_int8());
+        info.maxInstances = response[1].as_int8();
```

Comment:
Minor point here, but should probably use `as_uint8()` for all the parameters as they are `u8` in the MIP definitions

Reply:
Yeah, I think this was autocomplete that I didn't notice